pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -337,49 +347,52 @@ export const HandlerForm: React.FC<HandlerFormProps> = ({ id, endpointId }) => {
                         )}
                       </div>
                     </div>
+                    <div className="row">
+                      <div className="col-6">
+                        <TextareaGroup
+                          name={"conditions"}
+                          id={"conditionsInput"}
+                          defaultValue={handler?.conditions}
+                          required={true}
+                        />
```

Comment:
There are a couple of issues here.

1. The gateway expects a valid JSON to be sent within the `conditions` textarea, this should reflect in the frontend.
2. The frontend is showing success alerts while getting a `422` response from the gateway (this should be showing the error alert and message).